echopype/echodata/echodata.py

```diff
@@ -166,7 +164,7 @@ def _check_suffix(self, filepath):
         return suffix
 
     def _load_group(self, filepath, group=None):
```

Comment:
While you're at this, could you include type hints for the methods within Echodata objects also? Thanks! 😄